src/components/tabs/tabview.coffee

```diff
@@ -158,7 +158,7 @@ module.exports = class KDTabView extends KDScrollView
       else if firstPane = @getPaneByIndex 0
         @showPane firstPane
 
-    @emit "PaneRemoved", { pane, handle }
+    @emit "PaneRemoved", { pane, handle, isDetached: shouldDetach }
     return { pane, handle }
 
   removePaneByName:(name)->
```

Comment:
At first I thought, we shouldn't emit `PaneRemoved` when pane is detached. But I fear to break other things by not emitting an event. @sinan what do you think about it?